object_storage_api/core/exceptions.py

```diff
@@ -53,3 +60,26 @@ class InvalidImageFileError(BaseAPIException):
 
     status_code = 422
     response_detail = "File given is not a valid image"
+
+
+class MissingRecordError(DatabaseError):
+    """
+    A specific database record was requested but could not be found.
+    """
```

Comment:
```suggestion
    """A specific database record was requested but could not be found."""
```

Reply:
For the exception and schemas docstring formatting, I was copying the existing templates in the `object_storage_api`. @joelvdavies I can change the rest of the exception and schemas to 1 line docstrings as well, unless there was a reason they were set like this?

Reply:
No reason just copied from IMS, so yeah you can change them.